VectorMath/Vector3Float.cs

```diff
@@ -303,12 +292,10 @@ public float[] ToArray()
 		/// <summary>
 		/// Gets or sets an OpenTK.Vector2d with the X and Y components of this instance.
 		/// </summary>
+		[JsonIgnore]
```

Comment:
@larsbrubaker  - is there any reason to keep this Xy property? It's unused and throws is the setter is used